var/data/companies.json

```
@@ -124,6 +124,26 @@
       "Creabilis.",
       "Creabilis.com"
     ]
+    "employees": {
+      "jf-viguier": [
+        {
+          "startDate": "2002-05-01",
+          "endDate": ""
+        }
+      ],
+      "micka-fdz": [
+        {
+          "startDate": "2015-09-21",
+          "endDate": "2024-02-04"
+        }
+      ],
+      "LaBisquerie": [
+        {
+          "startDate": "2021-10-01",
+          "endDate": ""
+        }
+      ]
```

```suggestion
      ]
```

@micka-fdz & @LaBisquerie : Could make your own PRs, please ?